platform/jobs/edxPlatformAccessibilityMaster.groovy

```diff
@@ -134,8 +134,8 @@ secretMap.each { jobConfigs ->
                    }
                }
            }
-           shell('cd edx-platform')
-           shell('RUN_PA11YCRAWLER=1 ./scripts/accessibility-tests.sh')
+           shell('cd edx-platform \n' +
+                 'RUN_PA11YCRAWLER=1 ./scripts/accessibility-tests.sh')
```

Comment:
Sure! I had them in the same line because otherwise it was 2 separate build steps, but I can revert it :D.